code/src/NCModel/Blocks.ts

```diff
@@ -397,13 +397,13 @@ export class RootBlock extends NcBlock
             }
             break;
 
-            case 'Command':
+            case MessageType.Command:
             {
                 let msgCommand = JSON.parse(msg) as ProtoCommand;
                 socket.send(this.ProcessCommand(msgCommand).ToJson());
             }
             break;
-            case 'Heartbeat':
+            case MessageType.Heartbeat:
             {
                 let msgHeartbeat = JSON.parse(msg) as ProtoHeartbeat;
```

Comment:
Could add a default value with a simple log message indicating something wrong with the message. This would help implementors locate simple problems (before we have some support for message validation at the top).